tools/bbbsupervisor/bbbsupervisor.go

```diff
@@ -105,6 +126,64 @@ func (lf logFollower) follow() {
 	errWriter.Write([]byte(fmt.Sprintf("command %v unexpectedly exited", fullCmd)))
 }
 
+// follow implements follower interface by following Prometheus server and querying for values forever.
+func (pl prometheusFollower) follow() {
+	for {
+		fmt.Printf("prometheusFollower querying for %q\n", pl.expression)
+		pl.query()
+		time.Sleep(30000 * time.Millisecond)
```

Comment:
`30 * time.Second` is a bit easier to read

Reply:
Thanks, done.